docs/reference/controls/itemscontrol.md

```diff
@@ -118,7 +118,7 @@ The view resizes horizontally, but content is hidden when it is too high. This c
 <img src={example01Screenshot} alt="" />
 
 ## Example 02
-This example bind an observable collection of tile items to an items control. The ItemPanel is set to a canvas and we use a style to position the Tile within the canvas.
+This example binds an observable collection of Rectangles (which are bound to a tile VM) to an ItemsControl. The ItemPanel is set to a canvas and we use a style to position the Tile within the canvas.
```

Comment:
It's a minor thing but can you change all the control references to use inline code blocks? It makes it clearer you are referring to an actual type. EG `ItemsControl` instead of ItemsControl.